setuptools_scm/git.py

```diff
@@ -4,7 +4,7 @@
 import warnings
 
 FILES_COMMAND = 'git ls-files'
-DEFAULT_DESCRIBE = 'git describe --tags --long --match *.*'
+DEFAULT_DESCRIBE = 'git describe --tags --long --dirty --match *.*'
```

Comment:
does this work consistently, i very vaugely recall trouble with some older git versions that still may be in some distros